uint32 camera_id
geometry_msgs/Quaternion rotation
---
string status
